doc/examples/filters/plot_attribute_operators.py

```diff
@@ -0,0 +1,80 @@
+"""
+===================
+Attribute operators
+===================
+Attribute operators (or connected operators) is a family of contour preserving
+filtering operations in mathematical morphology. They can be implemented by
+max-trees, a compact hierarchical representation of the image.
+
+Here, we show how to use area and diameter openings.
+
+References
+----------
+.. [1] Salembier, P., Oliveras, A., & Garrido, L. (1998). Antiextensive
+       Connected Operators for Image and Sequence Processing.
+       IEEE Transactions on Image Processing, 7(4), 555-570.
```

Comment:
A quick comment. Please, could you add DOIs with the format DOI:XXX

Thanks!

Reply:
@sciunto is this format convention documented anywhere in our contributors guide? If not, could you please add it? =)

Reply:
This was supposed to be in #2616 which silently died in its eggshell.

Reply:
I have added DOIs to everything, except for one where I did not find the DOI.